pytorch_toolkit/image_classification/tests/export_tests_custom_classification.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import os
+
+from common.test_case import create_image_classification_export_test_case
+
+
+kwargs = dict(
+    problem_name='imagenet-classification',
+    ann_file='',
```

Comment:
what does it mean?

Reply:
ImageFolder dataset doesn't have any annotation, so this field is empty now. Since we decided to change the dataset format, I have to completely rework handling of annotation in this template and add corresponding features in deep-object-reid dataset.